noxfile.py

```diff
@@ -46,8 +46,9 @@ def tests(session):
     """Default unit test session."""
     # Install all test dependencies, then install this package in-place.
     path = "tests"
-    session.install("-r", "requirements-test.txt")
-    session.install("-e", ".")
+    session.install(".[test]")
+    session.install(".[network_modeling]")
```

Comment:
how is this not brought in by "."?

Reply:
`.` only installs what's in `install_requires` (e.g. the main module).